_maps/map_files/BoxStation/BoxStation.dmm

```diff
@@ -37604,6 +37603,9 @@
 	dir = 1
 	},
 /obj/structure/table/reinforced,
+/obj/item/device/slime_scanner{
+	step_x = 0
```

Comment:
dv

Reply:
I think I have to completely redo it in the map because I can't hand edit the file